domains/Zero.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "zerobun",
+        "email": "tareku2003@gmail.com"
+    },
+    "record": {
+        "URL": "zerobun.localcert.net"
```

I cannot reach this website also the record needs to be a CNAME.
```suggestion
        "CNAME": "zerobun.localcert.net"
```